fuse_core/test/test_async_publisher.cpp

```diff
@@ -45,7 +45,7 @@ class MyPublisher : public fuse_core::AsyncPublisher
 {
 public:
   MyPublisher()
-  : fuse_core::AsyncPublisher(0),
+  : fuse_core::AsyncPublisher(),
```

Comment:
nit, `1` to match the ROS 1 test: https://github.com/locusrobotics/fuse/blob/0461f779f9ac4defca6ecfd102a675add92bb3a9/fuse_core/test/test_async_publisher.cpp#L49